src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -401,7 +617,13 @@ impl<S: StateStore> SyncedKvLogStoreExecutor<S> {
                                             let update_vnode_bitmap = barrier
                                                 .as_update_vnode_bitmap(self.actor_context.id);
                                             let barrier_epoch = barrier.epoch;
+
+                                            let current_time = Instant::now();
                                             yield Message::Barrier(barrier);
+                                            self.metrics
```

Comment:
In current implementation we have to carefully ensure that we do this measurement in every yield point. We may make this logic of measuring back-pressure to be more general and simpler.

It can be a wrapper over any inner stream. When it implement `Stream`, it start a timer every time its `poll_next` returns ready, and record the time elpased between its next call on `poll_next`.

Reply:
Fixed